AutoNumber/Options/AutoNumberOptionsBuilder.cs

```diff
@@ -42,10 +42,10 @@ public AutoNumberOptionsBuilder UseStorageAccount(string connectionStringOrName)
             return this;
         }
 
-        public AutoNumberOptionsBuilder UseStorageAccount(CloudStorageAccount storageAccount)
+        public AutoNumberOptionsBuilder UseStorageAccount(BlobServiceClient blobServiceClient)

```

Comment:
I don't think "UseStorageAccount" is still meaningful here.  I'd suggest renaming it. Please also update readme.md

Reply:
Renamed to `UseBlobServiceClient`